test/controllers/ingredients_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class IngredientsControllerTest < ActionDispatch::IntegrationTest
+  test "should get new" do
```

Comment:
```suggestion
  test "should get index" do
```
Update the `new` action to an `index` action